.release-please-manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  ".": "0.3.0",
```

Comment:
are these numbers now random? Lets start from 0.1.0 or something that match our previous release history.

Reply:
No these are not random, this file is declaring our current state of versions. Taiko-mono is currently versioned as 0.3.0 (we've 3 git tags / binary releases), so this will bump it to 0.4.0 on merge. All packages will receive a minor bump from this PR.

I think this is okay, because then we have a smaller subset of commits in this changelog (from v0.3.0 to the tag applied at this commit) but if you prefer we start fresh with a v0.1.0 release of taiko-mono, then I suggest we delete the tags v0.1.0...v0.3.0. Either way is fine, let me know which you think is better. Do you think the white paper version is fine as well as stated here?